dev/create-release/create-release.sh

```diff
@@ -228,14 +228,14 @@ if [[ ! "$@" =~ --skip-package ]]; then
 
   # We increment the Zinc port each time to avoid OOM's and other craziness if multiple builds
   # share the same Zinc server.
-  make_binary_release "hadoop1" "-Psparkr -Phadoop-1 -Phive -Phive-thriftserver" "3030" &
-  make_binary_release "hadoop1-scala2.11" "-Psparkr -Phadoop-1 -Phive -Dscala-2.11" "3031" &
-  make_binary_release "cdh4" "-Psparkr -Phadoop-1 -Phive -Phive-thriftserver -Dhadoop.version=2.0.0-mr1-cdh4.2.0" "3032" &
-  make_binary_release "hadoop2.3" "-Psparkr -Phadoop-2.3 -Phive -Phive-thriftserver -Pyarn" "3033" &
-  make_binary_release "hadoop2.4" "-Psparkr -Phadoop-2.4 -Phive -Phive-thriftserver -Pyarn" "3034" &
-  make_binary_release "mapr3" "-Pmapr3 -Psparkr -Phive -Phive-thriftserver" "3035" &
-  make_binary_release "mapr4" "-Pmapr4 -Psparkr -Pyarn -Phive -Phive-thriftserver" "3036" &
-  make_binary_release "hadoop2.4-without-hive" "-Psparkr -Phadoop-2.4 -Pyarn" "3037" &
+  make_binary_release "hadoop1" "-Psparkr -Psparkr-docs -Phadoop-1 -Phive -Phive-thriftserver" "3030" &
```

Comment:
Hmm - I agree that the number of flags is getting out of hand, but there are a couple of things I tried to separate out in the SparkR build that can probably be simplified

The reason we need this change was that R has an inbuilt help system where you can access a [package's man pages](http://cran.r-project.org/doc/manuals/r-release/R-intro.html#Getting-help). Now these man pages use another R package (roxygen2) to be generated. I tried to keep the binary build separate from the man pages build so that users could build them separately if required.

Now another way to do this might be to check if roxygen2 is installed in the binary build script and just build the man pages if it exists. This might help us get rid of the profile `sparkr-docs` -- Does that sound like a better solution ? I will make a follow up PR today